feature_set_quickstart_guide/main/feature_set.py

```diff
@@ -0,0 +1,53 @@
+from qwak.feature_store.entities.entity import Entity
+from datetime import datetime
+from qwak.feature_store.feature_sets import batch
+from qwak.feature_store.feature_sets.transformations import SparkSqlTransformation
+
+
```

Comment:
I'd add a general comment of a file header about this feature set - What does this file do